### PR TITLE
Use Prometheus common logging

### PR DESCRIPTION
**Pull request by skidder**
The Prometheus log package is deprecated in favor of the common/log package.

### CODE REVIEW

File: cmd/prom2json/main.go [hunk 1]
```diff
@@ -19,10 +19,9 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/prometheus/log"
-
 	dto "github.com/prometheus/client_model/go"
 
+	"github.com/prometheus/common/log"
```
Comment (beorn7) [inline cmd/prom2json/main.go]: This line should be where it was before the change (we sort in blocks: stdlib, external, named external, internal, named internal).

File: prom2json.go [hunk 2]
```diff
@@ -9,9 +9,9 @@ import (
 
 	"github.com/matttproud/golang_protobuf_extensions/pbutil"
 	"github.com/prometheus/common/expfmt"
-	"github.com/prometheus/log"
 
 	dto "github.com/prometheus/client_model/go"
+	"github.com/prometheus/common/log"
```
Comment (beorn7) [inline prom2json.go]: Same as above: Leave this line in the block it was before.

**Comment by beorn7** [comment]
Vendoring also needs an update.

**Comment by beorn7** [comment]
@skidder are you still able to work on this?

**Comment by skidder** [comment]
@beorn7 Thank you for the feedback! I've reorganized the imports and updated vendored dependencies to use the latest in `master` of the `prometheus/common` repo.

**Comment by beorn7** [comment]
Great stuff. Thanks a lot.

**Pull request merged**